### DISCUSSION TITLE
Details about BndryRegisterT

### DISCUSSION THREAD

**Discussion by ankithadas** [AMReX-Codes/amrex#4086] [Q&A] [answered]
Dear AMReX team,

Could you please clarify the meanings of `in_rad`, `out_rad`, and `extend_rad` in the constructor of `BndryRegisterT`? For instance, if we have a box with coordinates (lo [0,0], hi [10, 10]), how would the boundary register box be shaped on the x-low face given values like `in_rad=2`, `out_rad=2`, and `extend_rad=5` (or any similar values)?

Kind regards,
Ankith

**Comment by WeiqunZhang** [comment, marked as answer]
There are `in_rad` layers of cells inside and `out_rad` layers of cells outside and `extend_rad` cells on each side of the transverse driction. So for your example, its lower corner is (-2,-5) and the upper corner is (1,15), if I get the arithmetics correctly.
